### PR TITLE
Added Andrew Schumaker

### PR DESCRIPTION
**Pull request by inquisitor-schumaker**
Added myself to the community folder.

### CODE REVIEW

File: community/people/aps/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+role: Graduate Student
+image: Andrew_Github.jpg
+email: aschumaker@wisc.edu
+phone: "218/4608408"
+city_state_zip: "Fitchburg, WI 53711"
+services:
+  github: inquisitor-schumaker
+---
+
+Andrew Schumaker (Shoe-maker) is a nuclear engineering graduate student at the University of Wisconsin-Madison. His undergraduate degree is in physics from the United States Military Academy, where he graduated in 2011. He will be researching nonproliferation.
```
Comment (MicahGale) [inline community/people/aps/index.md:13]: 1. Wisconsin —  Madison should use an em dash with spaces around the em dash.
2. Change "will be" to present tense.

File: community/people/aps/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: person
+title: Andrew Schumaker
+role: Graduate Student
+image: Andrew_Github.jpg
+email: aschumaker@wisc.edu
+phone: "218/4608408"
+city_state_zip: "Fitchburg, WI 53711"
```
Comment (MicahGale) [inline community/people/aps/index.md:8]: Use your office location; not home location.

File: community/people/aps/index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+services:
+  github: inquisitor-schumaker
```
Comment (MicahGale) [inline community/people/aps/index.md:10]: What, no X? :P

**Review by MicahGale — CHANGES_REQUESTED** [review]

File: community/people/aps/index.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+layout: person
+title: Andrew Schumaker
+role: Graduate Student
+image: Andrew_Github.jpg
+email: aschumaker@wisc.edu
+phone: "218/4608408"
```
Comment (MicahGale) [inline community/people/aps/index.md:7]: Also including personal phone (I assume) is probably not necessary, but up to personal preference.

**Comment by MicahGale** [comment]
I agree @gonuke, and thanks for doing the build because I deleted my build environment again.

**Comment by inquisitor-schumaker** [comment]
Changes per request

File: community/people/aps/index.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+layout: person
+title: Andrew Schumaker
+role: Graduate Student
+image: Andrew_Github.jpg
+email: aschumaker@wisc.edu
+city_state_zip: "443 ERB, Madison, WI"
+  github: inquisitor-schumaker
```
Comment (gonuke) [inline community/people/aps/index.md:8]: Your last change took out this necessary line

```suggestion
services:
  github: inquisitor-schumaker
```

**Review by gonuke — CHANGES_REQUESTED** [review]
Looks like you removed one line too many

**Comment by inquisitor-schumaker** [comment]
Is the other line required? Whatever it was I removed I figured I didn't need since there was no useful information on it.

Respectfully,

Andrew Schumaker
Office: ERB 443
Email: ***@***.***
Campus ID: 9084958108
________________________________

**Review by gonuke — APPROVED** [review]
Welcome to CNERG!

**Pull request merged**